packages/cli/__tests__/auth/__integration__/__snapshots__/cli.auth.apiml.integration.test.ts.snap

```diff
@@ -151,7 +151,7 @@ exports[`auth login/logout apiml help should display the logout help 1`] = `
       The type of token to get and use for the API. Omit this option to use the
       default token type, which is provided by 'zowe auth login'.
 
-      Allowed values: apimlAuthenticationToken, jwtToken, LtpaToken2
+      Allowed values: ^apimlAuthenticationToken.*, jwtToken, LtpaToken2
```

Comment:
Do we want to show regexes like this in the CLI help output? I think users are more likely to be familiar with wildcards which are simpler to read (e.g., `apimlAuthenticationToken*`)

I'm just curious about this, not requesting a change - because I assume it would require enhancing Imperative to support displaying "allowed values" in a different format.

Reply:
I'm afraid it would require a change in imperative to provide a nicer display for `allowableValues` 😋 
I don't mind opening an issue for this though 😋